objcgen/EqualsHelper.cs

```diff
@@ -4,10 +4,10 @@
 
 namespace ObjC {
 	public class EqualsHelper : MethodHelper{
+		public IKVM.Reflection.Type ParameterType { get; set; }
```

Comment:
you should add ` using Type = IKVM.Reflection.Type;` at the top, or this will end up in many places in the file.